lib/tara/installer.rb

```diff
@@ -63,6 +76,50 @@ def bundle_gems
       end
     end
 
+    def find_installed_gems
+      Bundler.with_clean_env do
+        definition = Bundler::Definition.build('lib/vendor/Gemfile', 'lib/vendor/Gemfile.lock', false)
+        @without_groups.each do |group|
+          definition.add_optional_group(group)
+        end
+        specs = definition.specs.map do |gem_spec|
+          spec = {
+            :name => gem_spec.name,
+            :full_name => gem_spec.full_name,
+            :full_gem_path => gem_spec.full_gem_path,
+            :spec_file => gem_spec.spec_file,
+            :bin_dir => gem_spec.bin_dir,
+          }
+          if spec[:full_gem_path].start_with?(Bundler.bundle_path.to_s) # Local gem
```

Comment:
Is there any case when this is actually `false`? For gems from the standard library perhaps?